docs/ACCESSIBILITY.md

```diff
@@ -115,6 +115,18 @@ When the "New messages" button is clicked:
 -  If a [tabbable](#tabbable) UI is found, focus on it
    -  Otherwise, focus on the send box wihout soft keyboard
 
+### ARIA role "separator"
+
+We are using the [ARIA role "separator"](https://www.w3.org/TR/wai-aria-1.1/#separator) for the "New messages" button. This is because the button serves as a visible boundary between read and unread messages, similar to an [horizontal ruler](https://www.w3.org/TR/html52/grouping-content.html#the-hr-element). Per [HTML 5.2 specifications](https://www.w3.org/TR/html52/grouping-content.html#the-li-element), separator role is allowed to use in the `<li>` element.
```

Comment:
```suggestion
We are using the [ARIA role "separator"](https://www.w3.org/TR/wai-aria-1.1/#separator) for the "New messages" button. This is because the button serves as a visible boundary between read and unread messages, similar to a [horizontal ruler](https://www.w3.org/TR/html52/grouping-content.html#the-hr-element). Per [HTML 5.2 specifications](https://www.w3.org/TR/html52/grouping-content.html#the-li-element), the separator role is allowed to be used in the `<li>` element.
```